common/source/docs/common-non-gps-navigation-landing-page.rst

```diff
@@ -14,6 +14,8 @@ These are the available options that allow a vehicle to estimate its position wi
 .. toctree::
     :maxdepth: 1
 
+    ARK ADIS16507 <common-ark-adis16507>
```

Comment:
I am not sure we should be putting IMUs, gyros, etc. in the AP wiki....these are board/subsystem components and definitely do not provide non-gps-navigation in and of themselves....they really do not fit anywhere I see in the AP wiki
I would suggest that ARK Elctronics be added here, however:https://ardupilot.org/ardupilot/docs/common-stores.html

Reply:
Maybe if I would add it below the note I have attached?
![Screenshot 2024-11-07 at 10-27-05 Non-GPS Navigation — Copter documentation](https://github.com/user-attachments/assets/ce972064-c86b-4078-be0a-97c00648308f)

If not I'll remove it. I'll add ARK as a shop as well after your feedback!

Reply:
I think it should just be removed....thanks